src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
We were waiting on a BuildTools update but we now have everything we need in dev/api to eliminate these older builds as their assets should just come from the last stable package we shipped.